wsd/Admin.cpp

```diff
@@ -170,6 +170,11 @@ void AdminSocketHandler::handleMessage(const std::vector<char> &payload)
             std::set<pid_t> pids = model.getDocumentPids();
             if (pids.find(pid) != pids.end())
             {
+                if (Admin::instance().logAdminAction())
+                {
+                    LOG_ANY("Admin request to kill document [" << model.getFilename(pid)
```

Comment:
We need to use this: COOLWSD::anonymizeUrl(getFilename())) to follow the logging anonymization logic properly I think.

Otherwise this is all good :-)

Reply:
Done